src/components/grid-view/GridView.scss

```diff
@@ -1,53 +1,31 @@
-@import '../../elements/common/variables';
-@import '../../styles/variables';
-
 .bdl-GridView {
-    padding: 0 5px;
-
-    &.bdl-GridView--columns-1 .bdl-GridView-slot {
-        $columns1prct: 100%;
-
-        flex-basis: $columns1prct;
-        max-width: $columns1prct;
+    .ReactVirtualized__Table__rowColumn:first-child {
+        margin: 0;
     }
 
-    &.bdl-GridView--columns-3 .bdl-GridView-slot {
-        $columns3prct: calc(33.3333% - 10px); /* 100%/3 - 10px */
-
-        flex-basis: $columns3prct;
-        max-width: $columns3prct;
+    & .bce-more-options {
+        align-self: flex-start;
     }
 
-    &.bdl-GridView--columns-5 .bdl-GridView-slot {
-        $columns5prct: calc(20% - 10px); /* 100%/5 - 10px */
-
-        flex-basis: $columns5prct;
-        max-width: $columns5prct;
+    & .bce-btn-more-options {
+        margin-top: 0;
+        padding-bottom: 5px;
+        padding-left: 10px;
+        padding-right: 10px;
+        padding-top: 4px;
```

Comment:
This looks like a magic number. Can we keep the padding consistent and use flexbox to vertically center the content?